src/foam/u2/crunch/CapabilityStore.js

```diff
@@ -467,14 +469,21 @@ foam.CLASS({
             this.IN('featured', this.Capability.KEYWORDS)
           )).select())
         .then(async sink => {
-          if ( sink.array.length == 1 ) {
+          if ( sink.array.length !== 1 )
+            return;
+
             let cap = sink.array[0];
             let ucj = await this.junctions.find(ucj => ucj.targetId == cap.id);
             if ( ucj && ( ucj.status == this.CapabilityJunctionStatus.GRANTED
               || ucj.status == this.CapabilityJunctionStatus.PENDING ) ) return;
 
-            this.openWizard(cap, false);
-          }
+            let x = this.ctrl.__subContext__;
+            let capa = await this.crunchService.updateJunction(x, cap.id, null, this.CapabilityJunctionStatus.GRANTED);
+
+            if ( capa.status != this.CapabilityJunctionStatus.GRANTED )
+              this.openWizard(cap, false);
+            else
+              this.window.location.reload();
```

Comment:
also this refresh isn't needed.... what is needed is : 

<img width="1030" alt="Screen Shot 2022-03-24 at 11 15 22 AM" src="https://user-images.githubusercontent.com/10802216/160912569-f3af9d2c-bc1c-4d48-a0f0-ea92f41e7508.png">

Reply:
[Anna]without changing the refreshWizarlet - wouldn't there be an error if you just signup a user in connect?
[Minsun Kim]for connect  sign-up and onboarding, treviso/cards  fine : )